internal/pkg/skuba/deployments/ssh/ssh.go

```diff
@@ -88,16 +91,22 @@ type Target struct {
 	targetName   string
 	sudo         bool
 	port         int
+	bastion      string
+	bastionUser  string
+	bastionPort  int
 	verboseLevel string
 	client       *ssh.Client
 }
 
 // GetFlags adds init flags bound to the config to the specified flagset
 func (t *Target) GetFlags() *flag.FlagSet {
 	flagSet := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
-	flagSet.StringVarP(&t.user, "user", "u", "root", "User identity used to connect to target")
+	flagSet.StringVarP(&t.bastionUser, "bastion-user", "", "", "User identity used to connect to the bastion using SSH (default to target user)")
+	flagSet.IntVarP(&t.bastionPort, "bastion-port", "", defSSHPort, "Port to connect to the bastion using SSH")
+	flagSet.StringVarP(&t.bastion, "bastion", "", "", "IP or FQDN of the bastion to connect to the other nodes using SSH")
+	flagSet.StringVarP(&t.user, "user", "u", "", "User identity used to connect to target using SSH")
```

Comment:
Because you remove the default of user but it's mandatory for any ssh-related commands, it's better to add a required check.

```
	_ = cobra.MarkFlagRequired(flagSet, "user")
```

Reply:
fixed with https://github.com/SUSE/skuba/pull/1185/commits/d6a20b613b7ad6a698db3000066e9bac4a825eaf